tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.Iast.cs

```diff
@@ -64,14 +64,14 @@ internal class Iast
             /// Default value is in TracerSettings
             /// </summary>
             /// <seealso cref="IastSettings.RedactionKeysRegex"/>
-            public const string RedactionKeysRegex = "DD_IAST_REDACTION_KEYS_REGEXP";
+            public const string RedactionKeysRegex = "DD_IAST_REDACTION_NAME_PATTERN";
```

Comment:
it's still a breaking chance as clients could have got the key name from the source. However, as IAST is still in beta, I think this is still okay, it would not be if it were GA.

We should still put a big warning in the release notes.